packages/contracts-bedrock/contracts/test/CrossDomainMessenger.t.sol

```diff
@@ -17,3 +23,100 @@ contract CrossDomainMessenger_BaseGas_Test is Messenger_Initializer {
         L1Messenger.baseGas(hex"ff", _minGasLimit);
     }
 }
+
+// CrossDomainMessenger_RelayMessage_Test tests re-entrency of relayMessage.
+contract CrossDomainMessenger_RelayMessage_Test is Messenger_Initializer {
+    // Storage slot of the l2Sender
+    uint256 constant senderSlotIndex = 50;
```

Comment:
Just had a though, i wonder if it makes sense to have a lib in `Common.t.sol` that includes constants for storage slots we want to set dynamically so in case they change we can update a single place

Reply:
Maybe it makes sense to pull slots + constants out further into their own contract? Something like `State.t.sol`?